controls/postgres_spec.rb

```diff
@@ -148,6 +86,15 @@
 end
 
 control 'postgres-05' do
+  impact 1.0
+  title 'Delete not required procedural languages'
+  desc 'You should delete programming languages which are not necessary. "internal", "c", "plpgsql" and "sql" are allowed defaults.'
+  describe postgres_session(USER, PASSWORD).query("SELECT COUNT(*) FROM pg_language where lanname NOT IN ('internal', 'c', 'sql', 'plpgsql');") do
```

Comment:
maybe we should add an `only_if` ondition, to only do this, when we have valid credentials? (also relevant for all other controls requiring credentials)

Reply:
I'm not sure, that this is useful in this baseline. There're a lot of controls which needs credentials. In my opinion it is mandatory to provide valid credentials to improve better security.

Reply:
Discussed and changed